server/mailet/mailets/pom.xml

```diff
@@ -161,6 +177,12 @@
             <artifactId>james-server-queue-memory</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-rate-limiter-redis</artifactId>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why?

Reply:
https://github.com/apache/james-project/pull/1093#discussion_r931785014

Reply:
Can you write a plain english response please? (link not working here.)

Reply:
> for RSpamDExtension, when without Redis dependency, it throws a RedisExtension not found

Reply:
Finally, it will be removed, because this mailet will be move  to third-party/rspamd